metaflow/event_logger.py

```diff
@@ -1,33 +1,21 @@
-from .sidecar import SidecarSubProcess
-from .sidecar_messages import Message, MessageTypes
+from metaflow.sidecar import Message, MessageTypes, Sidecar
 
 
-class NullEventLogger(object):
-    def __init__(self, *args, **kwargs):
-        pass
-
-    def start(self):
-        pass
-
+class BaseEventLogger(Sidecar):
```

Comment:
Can this be just `EventLogger`?

Reply:
Also, what necessitated changes in this class?

Reply:
It's more consistent with BaseMonitor but it's not really an issue. I alias it because for some reason it is exposed at the TL (I don't think *anyone* uses it since it is totally undocumented).